code/modules/cargo/bounties/assistant.dm

```diff
@@ -82,8 +82,8 @@
 /datum/bounty/item/assistant/donut
 	name = "Donuts"
 	description = "CentCom's security forces are facing heavy losses against the Syndicate. Ship donuts to raise morale."
-	reward = CARGO_CRATE_VALUE * 6
-	required_count = 10
+	reward = CARGO_CRATE_VALUE * 5
```

Comment:
Shouldn't the reward be increased instead? I would think the scarcity would make them more valuable, not less.

Reply:
Why is this a review? Does github not have a "comment on a specific line of code _without_ starting a review" function that doesn't go through the file view -> permalink line(s) interface?

Reply:
If you add the other line that was changed to your review, you would see that the required count was reduced to 6 to reflect the decreased reward.

Reply:
In other news I am blind.